lib/hackney/income/tenancy_gateway.rb

```diff
@@ -100,6 +100,7 @@ def get_tenancy(tenancy_ref:)
           t.primary_contact_name = tenancy['tenancy_details']['primary_contact_name']
           t.primary_contact_long_address = tenancy['tenancy_details']['primary_contact_long_address']
           t.primary_contact_postcode = tenancy['tenancy_details']['primary_contact_postcode']
+          t.pay_ref = tenancy['tenancy_details']['pay_ref']
```

Comment:
This will explode for 2 reasons
- the field does not exist
- the field will be called "payment_ref"
- the field is sometimes null anyway

```suggestion
          t.pay_ref = tenancy.dig('tenancy_details', 'payment_ref')
```

Reply:
This suggested change means this card can be merged now